### PR TITLE
add allow-port stage

### PR DESCRIPTION
**Pull request by mehabhalodiya**
Automate/Fix the 'port_allow' issue of aws-ocp in the Jenkins for the k8s netperf tests

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -81,6 +81,21 @@ pipeline {
           By default shares with email of person who ran the job
         '''
     )
+    string(
+        name: 'LEASED_RESOURCE',
+        defaultValue: '',
+        description: 'The cloud provider region'
+    )
+    string(
```
Comment (paigerube14) [inline Jenkinsfile:89]: Meha, what would this variable, you are passing here, expected to look like? All aws creds should be built into jenkins already that you shouldn't pass

Example of setting aws creds and region here: https://github.com/openshift-qe/ocp-qe-perfscale-ci/blob/b91f53beae5e98aacbf46cdc7ed4f3779d800b60/Jenkinsfile#L458-L461

**Comment by mehabhalodiya** [comment]
@qiliRedHat Can you please review it? Thanks!

File: Jenkinsfile [hunk 2]
```diff
@@ -73,6 +73,11 @@ pipeline {
           defaultValue: true,
           description: 'Boolean to create a google sheet with comparison data'
     )
+    booleanParam(
+          name: 'Network_Policy',
```
Comment (paigerube14) [inline Jenkinsfile:77]: lets set this to all caps for the name of the variable like other variables

**Comment by mehabhalodiya** [comment]
> Can you link a passing job here?

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/mbhalodi-e2e-benchmarking-multibranch-pipeline/job/network-perf-v2/32/console

**Comment by qiliRedHat** [comment]
@mehabhalodiya How did you fix the 'region: command not found' issue in you previous test?

**Comment by mehabhalodiya** [comment]
> @mehabhalodiya How did you fix the 'region: command not found' issue in you previous test?

I was earlier not doing [this way](https://github.com/openshift-qe/ocp-qe-perfscale-ci/pull/634#discussion_r1707297454) as Paige suggested. But then I did the correction.

**Comment by qiliRedHat** [comment]
@mehabhalodiya After you addressing the last comment from Paige, this pr can be merged.

**Comment by mehabhalodiya** [comment]
> @mehabhalodiya After you addressing the last comment from Paige, this pr can be merged.

Done @qiliRedHat

**Pull request merged**